File: app/src/main/java/com/example/c0782918_w2020_mad3125_fp/ui/SplashActivity.java
package com.example.c0782918_w2020_mad3125_fp.ui;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Message;

import com.example.c0782918_w2020_mad3125_fp.R;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        getSupportActionBar().hide();



    }

     Handler handler = new Handler(){

         @Override
         public void handleMessage(@NonNull Message msg) {
             super.handleMessage(msg);
            
         }
     };
}

File: app/src/main/java/com/example/c0782918_w2020_mad3125_fp/ui/SplashActivity.kt
package com.example.c0782918_w2020_mad3125_fp.ui

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Message
import androidx.appcompat.app.AppCompatActivity
import com.example.c0782918_w2020_mad3125_fp.R

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        supportActionBar!!.hide()
        handler.sendEmptyMessageDelayed(101, 5000)
    }

    var handler: Handler = object : Handler() {
        override fun handleMessage(msg: Message) {
            super.handleMessage(msg)
            if (msg.what == 101) {
                val intent = Intent(this@SplashActivity, LoginActivity::class.java)
                startActivity(intent)
                finish()
            }
        }
    }
}